packages/devtools_app/lib/src/screens/debugger/program_explorer.dart

```diff
@@ -313,7 +313,7 @@ class _FileExplorer extends StatefulWidget {
 }
 
 class _FileExplorerState extends State<_FileExplorer> with AutoDisposeMixin {
-  final ScrollController _scrollController = ScrollController();
+  late final ScrollController _scrollController;
```

Comment:
Should we have a lint to enforce a consistent style about whether to use `late final` for cases like this or `final`?

Reply:
Functionally, initializing the scroll controller in the class should have the same result as initializing the scroll controller in `initState` (i.e. both only happen once). However, it feels like more idiomatic flutter to initialize in `initState` and dispose in `dispose`, but I don't know if there is an official style guide entry for this.